### PR TITLE
UIREQ-995: Increase code coverage for Jest/RTL tests

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
Increase code coverage for Jest/RTL test

## Approach 

- Increase code coverage
- Add possible for run axe tests with example

## Refs
https://issues.folio.org/browse/UIREQ-995

### CODE REVIEW

File: src/views/constants.test.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    it('should call itemBarcode with correct props', () => {
+      render(formatter.itemBarcode(requiredData));
+
+      expect(ItemLink).toHaveBeenCalledWith(expect.objectContaining({
+        request: requiredData,
+      }), {});
+    });
+
+    it('should return requestDate', () => {
```
Comment (artem-blazhko) [inline src/views/constants.test.js:71]:
```suggestion
    it('should render requestDate', () => {
```

File: src/views/constants.test.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      render(formatter.requestDate(requiredData));
+
+      expect(screen.getByText(requiredData.requestDate)).toBeInTheDocument();
+    });
+
+    it('should return status', () => {
```
Comment (artem-blazhko) [inline src/views/constants.test.js:77]:
```suggestion
    it('should render status', () => {
```

File: src/views/constants.test.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+    it('should call getFullName with correct props', () => {
+      render(formatter.requester(requiredData));
+
+      expect(getFullName).toHaveBeenCalledWith(expect.objectContaining({
+        ...requiredData.requester,
+      }));
+    });
+
+    it('should call requesterBarcode with correct props', () => {
```
Comment (artem-blazhko) [inline src/views/constants.test.js:95]:
```suggestion
    it('should call "RequesterLink" with correct props', () => {
```

File: src/views/constants.test.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+      expect(RequesterLink).toHaveBeenCalledWith(expect.objectContaining({
+        request: requiredData,
+      }), {});
+    });
+
+    it('should return default patronGroup value', () => {
+      expect(formatter.patronGroup(requiredData)).toEqual(MISSING_VALUE_SYMBOL);
+    });
+
+    it('should return requestType', () => {
```
Comment (artem-blazhko) [inline src/views/constants.test.js:107]:
```suggestion
    it('should render requestType', () => {
```

Comment (Dmitriy-Litvinenko) [reply src/views/constants.test.js:107]: Fixed

File: src/views/constants.test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/views/constants.test.js:71]: Fixed

File: src/views/constants.test.js (hunk 2 above)
Comment (Dmitriy-Litvinenko) [reply src/views/constants.test.js:77]: Fixed

File: src/views/constants.test.js (hunk 3 above)
Comment (Dmitriy-Litvinenko) [reply src/views/constants.test.js:95]: Fixed

**Review by artem-blazhko — APPROVED** [review]

**Review by vashjs — APPROVED** [review]

**Pull request merged**